mundane-parse/src/main/scala/com/ambiata/mundane/parse/TextEscaping.scala

```diff
@@ -0,0 +1,113 @@
+package com.ambiata.mundane.parse
+
+import scalaz.Equal
+import scalaz.Scalaz._
+
+sealed trait TextEscaping {
+  import TextEscaping._
+
+  def render: String = this match {
+    case Delimited =>
+      "delimited"
+    case Escaped =>
+      "escaped"
+  }
+}
+
+object TextEscaping {
+
+  object Delimited extends TextEscaping
+  object Escaped extends TextEscaping
```

Comment:
I would drop and just have the escaping implementation here. Parsing snapshots/chords is a useful API but not here.